Automattic/wp-calypso
# Add the Universal Footer to /themes and /theme sections

#### Proposed Changes

* /themes: Integrate the Universal footer bar
* 1226-gh-Automattic/martech
* pduY5I-dl-p2

#### Testing Instructions



* Log out
* Navigate to /themes or /themes/free or /theme/archeo
* Ensure that you see the sitewide Footer Nav bars.
* The page should be unaffected if logged in

#### Pre-merge Checklist
